packages/core/admin/ee/server/seat-enforcement.js

```diff
@@ -0,0 +1,153 @@
+'use strict';
+
+const _ = require('lodash');
+
+const enableUsersToLicenseLimit = async (numberOfUsersToEnable) => {
+  const data = await strapi.db.query('strapi::ee-store').findOne({
```

Comment:
This is used elsewhere, you definitely want to extract this to a service method :slightly_smiling_face:

Reply:
Also, it could be useful to integrate the json parsing of the value to avoid duplicating the same code everywhere :eyes: